Fix wireless.h to use linux kernel header files
including uapi version of wireless.h

--- a/include/linux/wireless.h
+++ b/include/linux/wireless.h
@@ -18,16 +18,17 @@
 
 /***************************** INCLUDES *****************************/
 
-#if 0
+#if 1
 	#include <linux/types.h>		/* for __u* and __s* typedefs */
 	#include <linux/socket.h>		/* for "struct sockaddr" et al	*/
 	#include <linux/if.h>			/* for IFNAMSIZ and co... */
+	#include <linux/wireless.h>
+	#include <uapi/linux/wireless.h>
 #else
 	#define __user
 	/* typedef uint16_t	__u16; */
 	#include <sys/socket.h>			/* for "struct sockaddr" et al	*/
 	#include <net/if.h>			/* for IFNAMSIZ and co... */
-#endif
 
 /****************************** TYPES ******************************/
 #ifdef CONFIG_COMPAT
@@ -84,4 +85,5 @@ struct	iwreq {
 	union	iwreq_data	u;
 };
 
+#endif
 #endif	/* _LINUX_WIRELESS_H */
